static/app/components/gridEditable/styles.tsx

```diff
@@ -240,6 +240,7 @@ const GridStatusFloat = styled('div')`
   align-items: center;
   width: 100%;
   height: ${GRID_STATUS_MESSAGE_HEIGHT}px;
+  overflow: hidden;
```

Comment:
The empty status message is a bit larger than it's container.
<img width="707" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/d22aaf3b-02fa-4132-8c66-ace8d54dfdce">